### DISCUSSION TITLE
Unraid Community App: Unable to Access AdGuard Home WebUI After Successful Installation

### DISCUSSION THREAD

**Discussion by NeZios** [AdguardTeam/AdGuardHome#6370] [General]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Custom (please mention in the description)

### Installation

Docker

### Setup

On one machine

### AdGuard Home version

 v0.107.40

### Action

I've configured AdGuard Home on my Unraid server using Docker. The configuration included setting it up on the br0 network with a static IP address (192.168.1.15). The Docker run command used for the setup is as follows:

```
docker run
  -d
  --name='AdGuard-Home'
  --net='br0'
  --ip='192.168.1.15'
  -e TZ="Europe/Paris"
  -e HOST_OS="Unraid"
  -e HOST_HOSTNAME="FamilyBlanc"
  -e HOST_CONTAINERNAME="AdGuard-Home"
  -e 'TCP_PORT_3000'='3000'
  -e 'TCP_PORT_53'='53'
  -e 'UDP_PORT_53'='53'
  -l net.unraid.docker.managed=dockerman
  -l net.unraid.docker.webui='http://192.168.1.15:3000/'
  -l net.unraid.docker.icon='https://raw.githubusercontent.com/SiwatINC/unraid-ca-repository/master/icons/adguard.png'
  -v '/mnt/user/appdata/adguard/workingdir':'/opt/adguardhome/work':'rw'
  -v '/mnt/user/appdata/adguard/config':'/opt/adguardhome/conf':'rw' 'adguard/adguardh
```

### Expected result

I anticipated being able to access the AdGuard Home web user interface (webUI) upon successful installation. The logs indicated that the service was available at http://192.168.1.15:3000/ after the launch.

### Actual result

However, despite the successful installation, I am unable to access the webUI using the provided address (http://192.168.1.15:3000/). The logs confirm the availability of the service at the said address, but attempts to access it fail.

Additional context: I use Wireguard VPN for remote tunneled access to my Unraid server.

I'd greatly appreciate any guidance, troubleshooting steps, or advice to resolve this issue and gain access to the AdGuard Home webUI.

Thank you for your support.

### Additional information and/or screenshots

Using Unraid v6.12.4

**Comment by ainar-g** [comment]
Hello.  Are you using a custom container or our official one?

 >  The logs confirm the availability of the service at the said address, but attempts to access it fail.

To clarify, which logs do you mean here?  AdGuard Home's?  If so, have you enabled verbose logging?

**Comment by NeZios** [comment]
Hi @ainar-g
I am using the AdGuard Home on the Community App in Unraid 
![image](https://github.com/AdguardTeam/AdGuardHome/assets/38002539/b26a7b21-24f9-411c-9746-d8087d33f086)

Here is the logs from adguard home after successful installation :

```
2023/10/29 22:40:31.042937 [info] AdGuard Home, version v0.107.40
2023/10/29 22:40:31.043184 [info] This is the first time AdGuard Home is launched
2023/10/29 22:40:31.043191 [info] Checking if AdGuard Home has necessary permissions
2023/10/29 22:40:31.043384 [info] AdGuard Home can bind to port 53
2023/10/29 22:40:31.044418 [info] safesearch default: disabled
2023/10/29 22:40:31.044920 [info] Initializing auth module: /opt/adguardhome/work/data/sessions.db
2023/10/29 22:40:31.045617 [info] auth: initialized.  users:0  sessions:0
2023/10/29 22:40:31.045624 [info] AdGuard Home updates are disabled
2023/10/29 22:40:31.045626 [info] web: initializing
2023/10/29 22:40:31.045631 [info] This is the first launch of AdGuard Home, redirecting everything to /install.html 
2023/10/29 22:40:31.045638 [info] AdGuard Home is available at the following addresses:
2023/10/29 22:40:31.045858 [info] go to http://127.0.0.1:3000
2023/10/29 22:40:31.045864 [info] go to http://192.168.1.15:3000
```

Thanks for your help !

**Comment by ainar-g** [comment]
Ah, I see.  Since it's not the official AdGuard Home Docker image, I'll transfer that to Discussions.  You could also ask for help on the app maintainers' forums, if they have one.

**Reply by NeZios** [reply]
Wait, I think I have the official Adguard Home Docker Image as you can see here : 
![image](https://github.com/AdguardTeam/AdGuardHome/assets/38002539/9aed6a40-7310-4fee-b0e6-2d8f97c13b2d)

**Comment by NeZios** [comment]
Update: it works using host as network type. 
But I want to use br0 to have AdGuard home his own IP address on my local network 

Thanks for help.

**Comment by manicou537** [comment]
It's a little bit late, but I had the same problem and found a solution. I simply renamed the Working Directories and it worked right away.

**Reply by Jusaus** [reply]
Thanks! Worked like a charm, just added letter "s" to both working dirs and got it working.

**Reply by PhilippRoessner** [reply]
> It's a little bit late, but I had the same problem and found a solution. I simply renamed the Working Directories and it worked right away.

this also worked for me, I tried for more than one hour to access the WebUI. is this a Bug?
